Funcky.Test/Extensions/ChunkTest.cs

```diff
@@ -0,0 +1,71 @@
+using System.Collections.Generic;
+using Funcky.Extensions;
+using Xunit;
+
+namespace Funcky.Test.Extensions
+{
+    public class ChunkTest
+    {
+        [Fact]
+        public void GivenAnEmptyEnumerableChunkReturnsAnEmptyList()
+        {
+            List<int> numbers = new List<int>();
```

Comment:
```suggestion
            var numbers = new List<int>();
```

Reply:
You could also use `Enumerable.Empty<int>`

Reply:
Fixed in 2fbe2ca5